src/test/java/org/humancellatlas/ingest/project/wranglingpriority/WranglingStateTest.java

```diff
@@ -0,0 +1,44 @@
+package org.humancellatlas.ingest.project.wranglingpriority;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.humancellatlas.ingest.project.wranglingstate.WranglingState;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.json.JsonTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static junit.framework.TestCase.assertEquals;
+
+@JsonTest
+@RunWith(SpringRunner.class)
```

Comment:
This line forces the tests to use JUnit Vintage tests rather than the new JUnit Jupiter runner.
It can be removed.